### PR TITLE
Version bump 0.10.0

### PR DESCRIPTION
**Pull request by labbati**
### Description

Version bump for 0.10.0

### Readiness checklist
- [x] [Changelog entry](docs/changelog.md) added, if necessary
- [ ] Tests added for this feature/bug

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
 All notable changes to this project will be documented in this file - [read more](docs/changelog.md).
 
-## [Unreleased]
+## [UNRELEASED]
+
+## [0.10.0]
+
+**WARNING: THIS IS  A BREAKING CHANGE RELEASE**
+
+Refer to the [Migration Guide](UPGRADE-0.10.md)
```
Comment (pawelchcki) [inline CHANGELOG.md]: Great!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,23 @@
 # Changelog
 All notable changes to this project will be documented in this file - [read more](docs/changelog.md).
 
-## [Unreleased]
+## [UNRELEASED]
+
+## [0.10.0]
+
+**WARNING: THIS IS  A BREAKING CHANGE RELEASE**
```
Comment (pawelchcki) [inline CHANGELOG.md]: Would it be possible to add short description who will this breakage affect ? 
I think there is a non insignificant subset of users who will not be affected, lets not make them worry too much ?

Unless I'm mistaken ofcourse and everyone will have to perform some migration ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,23 @@
...
+
+Refer to the [Migration Guide](UPGRADE-0.10.md)
 
 ### Added
 - Request init hook configuration allowing running arbitrary code before actual request execution #175
+- Support OpenTracing without depending on it #193
```
Comment (pawelchcki) [inline CHANGELOG.md]: ❤️

File: CHANGELOG.md (hunk 2 above)
Comment (labbati) [reply CHANGELOG.md]: Everyone should perform some upgrade. but I will add.

Comment (labbati) [reply CHANGELOG.md]: good idea actually

**Pull request merged**